### PR TITLE
➕  Add storybook -css-modules-preset

### PR DESCRIPTION
**Pull request by M-Sieger**
- changed startbutton css

### CODE REVIEW

**Review by mrana17 — APPROVED** [review]
gj 💪

**Comment by SilviaBarcellini** [comment]
lgtm 👍

**Review by ghost — APPROVED** [review]
gj 💪

**Review by RonjaMeinecke — APPROVED** [review]
😀GJ

File: components/button/StartButton.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  function handleClick(e: { preventDefault: () => void }) {
+    e.preventDefault();
+  }
+
+  return (
+    <a href="#" onClick={handleClick}>
+      <button className={styles.start}>
+        <div className={styles.text}>Start</div>
+      </button>
+    </a>
```
Comment (m-wiemers) [inline components/button/StartButton.tsx]: You can be safer if you give props to the functions. With this you have more security and you will be suggested what the function needs

```suggestion
export type StartButtonProps = {
 label: string
 handleClick: MouseEventHandler<HTMLButtonElement>
}

function StartButton({ label, handleClick }: StartButtonProps) {
  function handleClick(e: { preventDefault: () => void }) {
    e.preventDefault();
  }

  return (
      <button className={styles.start} onClick={handleClick}>
        <div className={styles.text}>{label}</div>
      </button>
```

File: pages/index.tsx [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
       <Head>
-        <title>Create Next App</title>
+        <title>Momentum</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
       <main className={styles.main}>
-        <Button primary label="Primary" />
-        <Button primary={false} label="Secondary" />
+        <StartButton />
```
Comment (m-wiemers) [inline pages/index.tsx:17]:
```suggestion
        <StartButton label="your label" handleClick={handleClick} />
```

now can add the function handleClick Directly under Home

**Review by m-wiemers — COMMENTED** [review]
Really gj!

**Pull request merged**